spec/integration/util/windows/security_spec.rb

```diff
@@ -388,8 +387,8 @@ def grant_everyone_full_access(path)
 
         describe "#owner=" do
           it "should accept a user sid" do
-            winsec.set_owner(sids[:admin], path)
-            expect(winsec.get_owner(path)).to eq(sids[:admin])
+            winsec.set_owner(sids[:current_user], path)
+            expect(winsec.get_owner(path)).to eq(sids[:current_user])
```

Comment:
One potential issue is that if `set_owner` fails, then the file/dir may be owned by the current user, so `get_owner` may continue to pass. I think it's more robust to set the owner to an account that we know isn't the current owner.

Unfortunately, the current owner will depend on the default DACL for the user that is executing the tests. For regular user accounts (that are not running with elevated privileges), the default DACL will assign the owner to be the current user. For accounts like LocalSystem (which is what jenkins CI runs as), the owner will be `Administrators`:

```
irb(main):012:0* FileUtils.touch('c:\work\puppet\footest')
=> ["c:\\work\\puppet\\footest"]
irb(main):013:0> Puppet::Util::Windows::Security.get_security_descriptor('c:\work\puppet\footest')
=> BUILTIN\Administrators
WIN-QP47VOHA2P4\None
  BUILTIN\Users:(I)                             0x1f01ff
  NT AUTHORITY\SYSTEM:(I)                       0x1f01ff
  BUILTIN\Administrators:(I)                    0x1f01ff
```

Here the owner is `Administrators`, group is `None`, and there are 3 inherited access control entries.